language-plutus-core/src/Language/PlutusCore/Evaluation/Machine/ExBudgetingDefaults.hs

```diff
@@ -1,9 +1,14 @@
 {-# LANGUAGE TemplateHaskell #-}
 module Language.PlutusCore.Evaluation.Machine.ExBudgetingDefaults where
 
-import           Data.Aeson.THReader
 import           Language.PlutusCore.Evaluation.Machine.ExBudgeting
 
+-- import           Data.Aeson.THReader
```

Comment:
Commented out code

Reply:
That one's here for a reason :-)

Reply:
... I mean I could make the TH nicer and emit a warning and a default when the model can't be loaded, but that does sound like some work. And I'm not even sure that's good, as if someone loads in the repo and gets a weird json for whatever reason, ignores the warning and ends up with an absolutely weird cost-model.